.eslintrc.json

```diff
@@ -1,3 +1,9 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": "next/core-web-vitals",
+  "rules": {
+    "no-console": "warn", // Disallow the use of console.log and console.error
+    "no-unused-vars": "error", // Disallow unused variables
+    "indent": ["error", 2], // Enforce 2 spaces for indentation
```

Comment:
Is this just a regular tab?

Reply:
Yes.

Reply:
@jovanpet I thought a normal tab was 4 spaces? I usually work with code bases that enforce a 2 spaces tab with Typescript though.

Reply:
I think we should remove that rule. I just let VSCode auto-format for me. I believe 4 spaces is my usual, but I haven't typed Ts in a while.

Reply:
If you use the ESLint extension as your formatter in VSCode, I believe that it will automatically adhere to rules in the `eslintrc.json` file and highlight violations. I could be wrong though.

Reply:
It should be 4 tho. I just checked the official styling guide.

Reply:
I think it should be 4 spaces for any ts file. I just read that online. But this is fine too.

Reply:
Make it 4 and see what happens.

Reply:
Gotcha, I'll make that change!

Reply:
Yess! I should also check and see if I can block prs that don't pass the workflow.

Reply:
I was wondering how to do that! I don't remember if an action is automagically rejected if a linter shows errors, or if we have to configure that ourselves...

Reply:
I think it needs to be pushed to the main branch before I change that?

Reply:
@johan253  I made the switch to 4. Looks better.